src/components/dashboard/Settings.js

```diff
@@ -107,14 +91,13 @@ const Settings = ({ screenProps, styles, theme, navigation }) => {
     (newValue, options) => {
       const { fromClaim = false } = options || {}
 
-      if (newValue && !allowedNotificationPermissions) {
-        requestNotificationPermissions(fromClaim ? { promptPopup: false } : {})
-        return
+      if (newValue === true) {
+        userProperties.setLocal('askedPermissionsAfterClaim', true)
```

Comment:
i think this should be set in Claim page just after we confirmed/dismissed dialog

Reply:
also no more needs allowed in deps

Reply:
done